plothist/plotters.py

```diff
@@ -598,6 +551,7 @@ def plot_comparison(
     comparison_ylabel=None,
     comparison_ylim=None,
     ratio_uncertainty="uncorrelated",
```

Comment:
Add this to a `kwargs`? I find it strange to have it as an argument if it only impact 2/4 comparison method.

Reply:
discussed in meeting